hacklab_ry.md

```diff
@@ -37,3 +39,7 @@ Esimerkiksi
 > Yhdistyksen/hallituksen kokous kannatti [äänestyksessä] Hacklab ry.:n varsinaisen jäsenyyden hakemista (y-tunnus 3084371-1) sekä liiton sääntöjen ja tarkoituksen hyväksymistä.
 
 Mikäli yhdistyksen säännöt tai yhdistyksen kokouksen muu päätös ei toisin määrää, liittymispäätöksen voi tehdä yhdistyksen hallitus. Liiton suositus on tehdä päätös yhdistyksen kokouksessa. Jäsenyyden hakemisesta on syytä mainita kokouksen kutsussa ja tarjota linkki [liiton sääntöihin](/saannot.html), jotta osallistujilla on ollut riittävästi aikaa tutustua niihin ennen päätöstä.
+
+## Tilinumero
+
+IBAN-tilinumeromme on FI45 5670 0820 3659 12. Otamme vastaan lahjoituksia. Mikäli teet lahjoituksen, ilmoitathan siitä hallitukselle.
```

Comment:
Figured an reference number for donations, `RF43 3800 1`. Also I'd change "ilmoitathan siitä hallitukselle" into some proper way of wishing (but not mandatory) "Toivomme ilmoitusta", all lawfully related data is in banking data, we can assume sender is donator if no other contact made.

Reply:
Also "anonymous donation" can be an thing, not that sender still has someones bank account :D but anyway I feel this wording is important in general, hence stalling for this =)

Reply:
changed the wording in 93081dc